pkg/apis/defaults/defaults.go

```diff
@@ -52,3 +55,13 @@ func setDefaultIfEmpty(attr map[string]string, k, v string) {
 		attr[string(k)] = v
 	}
 }
+
+func defaultKeyUsages() string {
+	var defKU []string
+
+	for _, ku := range cmapi.DefaultKeyUsages() {
```

Comment:
I think we're better to explicitly state defaults here - this will make it difficult for us to track changes over time if any more do happen (even if they are unlikely)

Reply:
Fair enough, I'll hard code them in